offer.html

```diff
@@ -56,7 +56,7 @@
   <div class="company">
     <div class="flex-container">
       <div class="company__info">
-        <h2 class="offer__header">Oferta</h2>
+        <span class="offer__header">Oferta</span>
```

Comment:
jak robię <h3> to mi za bardzo ucieka na dół, nie bardzo mi wychodzi w CSS ułożenie tego tak jak bym chciał.